File: java-spring-boot-gradle-hello/src/main/java/com/example/loanhello/model/HealthResponse.java
package com.example.loanhello.model;

import io.swagger.v3.oas.annotations.media.Schema;

@Schema(description = "Health check response")
public record HealthResponse(
        @Schema(example = "ok") String status,
        @Schema(example = "loan-service") String service,
        @Schema(example = "1.2.0") String version
) {}

File: kotlin-spring-boot-hello/bin/main/com/example/loanhello/model/HealthResponse.kt
package com.example.loanhello.model

import io.swagger.v3.oas.annotations.media.Schema

@Schema(description = "Health check response")
data class HealthResponse(
    @Schema(example = "ok") val status: String,
    @Schema(example = "loan-service") val service: String,
    @Schema(example = "1.2.0") val version: String
)
